apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -282,6 +282,21 @@ function load_block_patterns_from_api( $current_screen ) {
 }
 add_action( 'current_screen', __NAMESPACE__ . '\load_block_patterns_from_api' );
 
+/**
+ * Load WPCOM Block Patterns Modifications
+ *
+ * This is responsible for modifying how block patterns behave in the editor,
+ * including adding support for premium block patterns. The patterns themselves
+ * are loaded via load_block_patterns_from_api.
+ */
+function load_wpcom_block_patterns_modifications() {
+	if ( apply_filters( 'a8c_enable_block_patterns_modifications', true ) ) {
```

Comment:
I like the pink dot, too, but I think I'd default to leaving this to `false` so that we can deploy the feature switched off and then safely switch it on programmatically, rather than the other way around.